packages/examples/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this npm module are documented in this file.
 
+## 2023-09-11
+
+- Adjust to underlying api changes (`monaco-vscode-api` and `monaco-languageclient`)
+- Fix json language server lauch
```

Comment:
Typo: launch

Reply:
Well spotted. I will also change the date.

Reply:
Done ⬇️